source/operations/install-deploy-manage/expand-minio-deployment.rst

```diff
@@ -256,6 +256,8 @@ those appropriate for your deployment.
 Review the :ref:`expand-minio-distributed-prereqs` before starting this
 procedure.
 
+Complete any planned hardware expansion prior to :ref:`decommissioning older hardware pools <minio-decommissioning>`.
```

Comment:
This is a good graf to use on the decommissioning page as a really simple and to-the-point guide

Reply:
👍